packages/neutron-sdk/src/interchain_queries/queries.rs

```diff
@@ -103,6 +135,91 @@ pub fn query_balance(
     })
 }
 
+/// Returns bank total supply on remote chain for particular denom
+/// * ***registered_query_id*** is an identifier of the corresponding registered interchain query
+pub fn query_bank_total(
+    deps: Deps<InterchainQueries>,
+    _env: Env,
+    registered_query_id: u64,
+) -> NeutronResult<TotalSupplyResponse> {
+    let registered_query = get_registered_query(deps, registered_query_id)?;
+
+    check_query_type(registered_query.registered_query.query_type, QueryType::KV)?;
+
+    let total_supply: TotalSupply = query_kv_result(deps, registered_query.registered_query.id)?;
```

Comment:
```suggestion
    let total_supply: TotalSupply = query_kv_result(deps, registered_query_id)?;
```

And you could do that throughout all following functions in this file.